src/backend/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.0.0",
   "scripts": {
     "dev": "yarn nodemon index.ts",
-    "test": "vitest",
+    "test": "vitest --changed",
```

Comment:
what's this about?

Reply:
this enforces rerunning tests correctly when theres a file change

Reply:
ohh okay, i'll keep that then